oiosaml/src/main/java/dk/gov/oio/saml/session/SessionCleanerService.java

```diff
@@ -0,0 +1,64 @@
+package dk.gov.oio.saml.session;
+
+import dk.gov.oio.saml.config.Configuration;
+import dk.gov.oio.saml.service.OIOSAML3Service;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpSession;
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class SessionCleanerService {
```

Comment:
I miss Javadoc comments on many of the new classes at least briefly describing the intent.

Reply:
I have added javadoc to classes in the session package